site/content/blog/introducing-ckan-mcp-server-connect-ai-assistants-to-your-data-portals.md

```diff
@@ -0,0 +1,219 @@
+
+### The MCP Solution
+
+MCP replaces these fragmented integrations with a single, open protocol. Think of it as the "USB standard" for AI-data connections - any MCP-compliant data source can serve context to any MCP-enabled AI client, and vice versa.
+
+## Industry Adoption: The New Standard for AI
+
+The adoption of MCP has been remarkable. Major tech companies are rapidly integrating it:
+
+- **OpenAI** officially adopted MCP in March 2025, integrating it across ChatGPT, their Agents SDK, and Responses API
+- **Google DeepMind** announced MCP support in Gemini models, with CEO Demis Hassabis calling it "rapidly becoming an open standard for the AI agentic era"
+- **Microsoft** unveiled plans to make MCP a "foundational layer for secure, interoperable agentic computing" in Windows 11
+
+By early 2025, the community had already created over 1,000 MCP servers, covering everything from GitHub and Slack to databases and enterprise systems.
+
+## Introducing the CKAN MCP Server
+
+### What is CKAN?
+
+CKAN (Comprehensive Knowledge Archive Network) is the world's leading open-source data management system, used by governments, research institutions, and organizations worldwide to publish, share, and manage datasets. From national open data portals to research repositories, CKAN powers some of the most important data infrastructure on the web.
+
+### Why Connect CKAN to AI Assistants?
+
+Imagine being able to:
+
+- **Ask natural language questions** about datasets: "Show me environmental datasets from 2020"
+- **Get instant summaries** of complex data without browsing through catalogs
+- **Discover relationships** between datasets across different organizations
+- **Access metadata** and resources through conversational interfaces
+- **Integrate data discovery** into your existing AI workflows
+
+This is exactly what the CKAN MCP Server enables.
+
+## How It Works: Technical Overview
+
+### Architecture
+
+![MCP Server Architecture Diagram](/static/img/blog/introducing-ckan-mcp-server-connect-ai-assistants-to-your-data-portals/mcp-server-diagram.png)
+
+The CKAN MCP Server acts as a bridge between MCP-compatible AI clients and CKAN APIs. Here's the flow:
+
+1. **AI Assistant** sends a request through MCP protocol
+2. **CKAN MCP Server** translates the request to CKAN API calls
+3. **CKAN Portal** returns data and metadata
+4. **Server processes and formats** the response for the AI
+5. **AI Assistant** receives structured data to provide intelligent responses
+
+### Available Tools
+
+The server currently provides these MCP tools:
+
+- **`search_datasets`**: Search for datasets using natural language queries
+- **`get_dataset`**: Retrieve detailed information about specific datasets
+- **`list_organizations`**: Browse organizations and their data offerings
+
+## Deployment Options: Flexible and Scalable
+
+### Local Development
+Perfect for testing and development:
+
+```bash
+# Quick start with Docker
+docker-compose up -d
```

Comment:
I don't think reader can run this command without codebase (dockerfile etc.) so why are we providing these instructions?